api/src/main/java/org/openmrs/PersonAttribute.java

```diff
@@ -296,14 +296,26 @@ public void voidAttribute(String reason) {
 	 * @should return negative if this attribute has lower attribute type than argument
 	 * @should return negative if other attribute has lower value
 	 * @should return negative if this attribute has lower attribute id than argument
-	 * @should not throw exception if attribute type is null
-	 * @deprecated since 1.12. Use DefaultComparator instead.
-	 * Note: this comparator imposes orderings that are inconsistent with equals.
 	 */
-	@SuppressWarnings("squid:S1210")
 	public int compareTo(PersonAttribute other) {
-		DefaultComparator paDComparator = new DefaultComparator();
-		return paDComparator.compare(this, other);
```

Comment:
Aren't you just supposed to remove the deprecated annotation and not change the method code? Because it was already delegating to the the DefaultComparator anyways, i think this deprecation was invalid